external/header.php

```diff
@@ -130,7 +130,9 @@ function () {
         $time = array_key_exists('REQUEST_TIME', $_SERVER)
             ? $_SERVER['REQUEST_TIME']
             : time();
-        $requestTimeFloat = explode('.', $_SERVER['REQUEST_TIME_FLOAT']);
+        // In some cases there is comma instead of dot
+        $delimiter = (strpos($_SERVER['REQUEST_TIME_FLOAT'], ',') !== false) ? ',' : '.';
```

Comment:
What version of PHP are you using that emits a `,`?

Reply:
Yes, i'm from Ukraine. Your code show me 3.14159. There is `,` only in one project. But if there is any chance that it will be `,` instead of `.` then my code is useful =)